spec/spec.md

```diff
@@ -606,149 +606,151 @@ In the following message structure, the issuer wants a Permanent Resident Card (
                   "challenge":"508adef4-b8e0-4edf-a53d-a260371c1423",
                   "domain":"9rf25a28rs96"
                },
-               "id":"dcc75a16-19f5-4273-84ce-4da69ee2b7fe",
-               "version":"0.1.0",
-               "issuer":{
-                  "id":"did:example:123?linked-domains=3",
-                  "name":"Washington State Government",
-                  "styles":{
-                     
-                  }
-               },
-               "presentation_definition":{
-                  "id":"8246867e-fdce-48de-a825-9d84ec16c6c9",
-                  "frame":{
-                     "@context":[
-                        "https://www.w3.org/2018/credentials/v1",
-                        "https://w3id.org/citizenship/v1",
-                        "https://w3id.org/security/suites/bls12381-2020/v1"
-                     ],
-                     "type":[
-                        "VerifiableCredential",
-                        "PermanentResidentCard"
-                     ],
-                     "credentialSubject":{
-                        "@explicit":true,
+               "credential_manifest":{
+                  "id":"dcc75a16-19f5-4273-84ce-4da69ee2b7fe",
+                  "version":"0.1.0",
+                  "issuer":{
+                     "id":"did:example:123?linked-domains=3",
+                     "name":"Washington State Government",
+                     "styles":{
+                        
```

Comment:
is there a non empty object we can supply here?

Reply:
[updated](https://github.com/decentralized-identity/waci-presentation-exchange/compare/49ef725a91640149a855098944f6f56ca2d66351..f4de63446640af60cddcf1c4c6ac69b97dc5269d)